tests/test_web_sendfile_functional.py

```diff
@@ -288,13 +308,12 @@ async def handler(request):
     client = await aiohttp_client(app)
 
     resp = await client.get("/")
-    assert 200 == resp.status
```

Comment:
Why did you invert the operands? Please, revert. Unnecessary changes only create complications with backporting.

Reply:
Because different tests had them in a different order for identical assertions, which was confusing.  I altered one and copied to the others.  I was debugging with all 3 tests.

Reply:
Alright. This still needs to be reverted so the diff is smaller and is less prone to merge conflicts on cherry-picks.